velox/connectors/Connector.cpp

```diff
@@ -86,7 +86,7 @@ void Connector::unregisterTracker(cache::ScanTracker* tracker) {
 std::shared_ptr<cache::ScanTracker> Connector::getTracker(
     const std::string& scanId,
     int32_t loadQuantum) {
-  return trackers_.withWLock([&](auto& trackers) -> auto {
+  return trackers_.withWLock([&](auto& trackers) -> auto{
```

Comment:
Why remove the space after 'auto'? BTW, why do we need -> auto at all?

Reply:
clang format does so. Let me double check.